overlord/devicestate/handlers_install.go

```diff
@@ -144,3 +150,21 @@ func checkEncryption(model *asserts.Model) (res bool, err error) {
 
 	return true, nil
 }
+
+// configureRunSystem configures the ubuntu-data partition with any
+// configuration needed from e.g. the gadget or for cloud-init
+func configureRunSystem() error {
```

Comment:
as we talked about this should probably live in a toplevel sysconfig  package so that we can reuse from snap-bootstrap